backends/xnnpack/CMakeLists.txt

```diff
@@ -72,6 +72,7 @@ target_include_directories(
   xnnpack_schema INTERFACE ${_xnnpack_schema__include_dir}
                            ${EXECUTORCH_ROOT}/third-party/flatbuffers/include)
 
+target_compile_options(pthreadpool PUBLIC ${_common_compile_options})
```

Comment:
Why is this needed? and pthreadpool related stuff has moved to root CMakeLists

Reply:
my local build failed complaining no -fPIC